PENDING.md

```diff
@@ -54,6 +54,7 @@ FEATURES
 * [store] Add transient store
 * [gov] Add slashing for validators who do not vote on a proposal
 * [cli] added `gov query-proposals` command to CLI. Can filter by `depositer`, `voter`, and `status`
+  * Added staking query cli cmds for unbonding-delegations and redelegations
```

Comment:
Should we really adding this under the top line? Should be it's separate entry with the referenced PR (or issue -- we need to come to a consensus on that).

Reply:
Idk, I was just following the examples from above, but I'll change if you think it's better that way